pkg/kapp/diffgraph/change_graph_test.go

```diff
@@ -465,6 +469,73 @@ metadata:
 	}
 }
 
+func TestChangeGraphWithChangeGroupPlaceholderInConfDefaults(t *testing.T) {
+	configYAML := `
+apiVersion: v1
```

Comment:
i think we need to have multiple namespaces and multiple CRDs to see that the relationship is granular. otherwise it's not different from today.

Reply:
Sure, I will add more namespaces and CRDs

Reply:
Done

Reply:
@cppforlife Added more namespaces, namespaced resources, CRDs and CRs.